Automattic/sensei
# Preserve status when emails are re-created

Resolves #6893.

## Proposed Changes
- Preserve the existing status of emails when they are re-created.
- Also changes "recreate" (incorrect) to "re-create" (correct).

## Testing Instructions


1. Disable some of the emails and enable others.
2. Re-create the emails from Sensei > Tools.
3. Check to ensure that the status of every email is preserved.

## Pre-Merge Checklist
